lbrynet/daemon/Daemon.py

```diff
@@ -939,6 +951,33 @@ def _get_lbry_files(self, return_json=False, full_status=True, **kwargs):
         log.debug("Collected %i lbry files", len(lbry_files))
         defer.returnValue(lbry_files)
 
+    def _sort_lbry_files(self, lbry_files, sorts):
+        for field, order in sorts:
+            reverse = order == FILE_SORT_ORDER_DESCENDING
+            if field == FILE_SORT_FIELD_NAME:
+                lbry_files = sorted(lbry_files, key=lambda f: f.file_name, reverse=reverse)
+            elif field == FILE_SORT_FIELD_DATE:
+                lbry_files = sorted(lbry_files, reverse=reverse)
+            elif field == FILE_SORT_FIELD_PRICE:
+                lbry_files = sorted(lbry_files, key=lambda f: f.points_paid, reverse=reverse)
+            else:
+                raise Exception('Unrecognized sort field "{}"'.format(field))
+        return lbry_files
```

Comment:
Instead of having the variable `lbry_files` I think it's fine to just have `return` in front of each `sorted()` call. Less code, equally easy to read, less variables, etc.

Reply:
The main reason I did this was to support multiple passes, so for example

    lbrynet-cli file_list --sort=name,desc --sort=price,asc

would produce a list sorted by price, but items with the same price would be sorted by name.